### PR TITLE
chat/usermanager

### PR DESCRIPTION
**Pull request by louiserplh**

### CODE REVIEW

**Review by natgate — APPROVED** [review]
Code looks good to me! 
Good job to have refactored the call to the database!

File: app/src/main/java/ch/epfl/sweng/hostme/ui/messages/ChatActivity.java [hunk 1]
```diff
@@ -48,12 +47,15 @@ public class ChatActivity extends AppCompatActivity {
     private ChatAdapter chatAdapter;
     private ImageView launchButt;
     private String conversionId = null;
```
Comment (Scott2Williams) [inline app/src/main/java/ch/epfl/sweng/hostme/ui/messages/ChatActivity.java:49]: I was a little confused by everything in the code being labeled `conversion...` but I quickly realized that it means "conversation". Perhaps we will take a quick moment to change that to `conversation...` or `convo...` for readability in a future PR.

**Review by Scott2Williams — APPROVED** [review]
Nice, I like how a lot of the data is localized into the `Preferences` map. That will do wonders for performance :)

**Review by jules20022000 — APPROVED** [review]
Your call to the database is cleaner as before and all the rest make sense.

**Review by paul-ndl — APPROVED** [review]
I don't understand what is this code about but I don't see any problem in the code except some typo

**Pull request merged**